src/lib/apiTypesV2/HttpInfo.cpp

```diff
@@ -91,4 +94,52 @@ std::string HttpInfo::toJson()
   return jh.str();
 }
 
+
+
+/* ****************************************************************************
+*
+* HttpInfo::fill -
+*/
```

Comment:
This is a style-guide issue.
Personally, I don't like the 'using', as information is 'removed' somehow.
When mongo:: (well, any namespace, like 'std::', for example) is present, it is 100% clear where the type came from.
Using `using namespace` it becomes a whole less clearer.
At the same time, it is true that the code gets shorter and 'easier' to read.
But, easier to read only if you know where each type comes from.
So, not easier to read, but more cryptic, IMHO.

Let's discuss this, but my opinion is definitely that namespaces should always be written out.